app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -66,9 +84,26 @@ angular.module('miq.util').factory('chartsMixin', ['pfUtils', function(pfUtils)
       dataName : __('KBps'),
       tooltipFn  : dailyTimeTooltip
     },
+    hourlyNetworkUsageConfig: {
+      chartId  : 'networkUsageHourlyChart',
+      headTitle: __('Network Utilization Trend'),
+      timeFrame: __('Last 24 Hours'),
+      units    : __('KBps'),
+      dataName : __('KBps'),
+      tooltipFn  : hourlyTimeTooltip
+    },
+    realtimeNetworkUsageConfig: {
```

Comment:
can we use here the same `NetworkUsageConfig` struct and just change `timeFrame` when we know witch one we use ?